daos-stack/daos
# DAOS-15198 test: Test DAOS_PROP_PO_SPACE_RB - Pool space reservation

Test steps:
1. Create pool with space_rb set to 50 and aggregation disabled.
2. Call dmg pool get-prop and verify that Rebuild space ratio (space_rb) is 50%.
3. Run IOR to fill 50% of SCM. Use small transfer size so that data are written to SCM. IOR should fail with DER_NOSPACE(-1007): 'No space left on device'
4. Create a new pool with space_rb set to 50 and aggregation enabled this time.
5. Call dmg pool get-prop and verify that Rebuild space ratio (space_rb) is 50%.
6. Run IOR to fill 50% of NVMe. Use large transfer size so that data are written to NVMe. IOR should fail with DER_NOSPACE(-1007): 'No space left on device'

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: test_space_rb

### Before requesting gatekeeper:


### Gatekeeper:

* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
